skaffold.yaml

```diff
@@ -1,3 +1,14 @@
+.aea_version: &aea_version
+  AEA_VERSION: "1.17.0"
+  AUTHOR: "valory"
+.docker_args: &docker_args
+  dockerfile: Dockerfile
+.release_docker_args: &release_docker_args
+  <<: *docker_args
+  noCache: true
+.open_autonomy_runtime_context: &open_autonomy_runtime_context
```

Comment:
We have 2 open-autonomy images in different places. This confused me. 
@angrybayblade Why is this?

Reply:
`We have 2 open-autonomy images in different places. This confused me.` where?